# 唯一失敗するべきサンプル。
func main
  hoge = 4
  a(4)
endfunc

func a(x);test = x;endfunc